internal/server/middleware_test.go

```diff
@@ -359,36 +359,15 @@ func TestOrgLookupByDomain(t *testing.T) {
 		})
 
 	req := httptest.NewRequest("GET", "/foo", nil)
-	req.Header.Add("Host", "umbrella.infrahq.com")
+	req.Host = "umbrella.infrahq.com"
 
 	resp := httptest.NewRecorder()
 	router.ServeHTTP(resp, req)
 
 	assert.Equal(t, resp.Code, 200)
 }
 
-func TestMissingOrgErrors(t *testing.T) {
```

Comment:
This test was removed because this case is no longer possible now that we're using `req.Host` to get the host header.

In the future we'll restore this test case when we change the behaviour to be configurable, and  return fake data instead of the default org.

I'd also like to change these tests to use a real http server and client. That way the request is the server receives is actually parsed as it would be in production, instead of us attempting to create an `http.Request` that may or may not match production.

Reply:
I think we need these test changes to confirm this fix, so going to work on that now.

Reply:
seems like this is still a valid use case for saas.

Reply:
Ya, we'll re-introduce this behaviour with the config setting